challenges/type-aliases/tests.ts

```diff
@@ -0,0 +1,41 @@
+import { Expect, Equal } from 'type-testing';
+
+interface Spacecraft {
+  name: Name;
+  yearBuilt: Year;
+  crewCapacity: Count;
+  launchDate: Date;
+  isOperational: IsOperational;
+  propulsionSystem: string[];
+  payload: Payload[];
+}
+
+type test_Name = Expect<Equal<Name, string>>;
+type test_Year = Expect<Equal<Year, number>>;
```

Comment:
this challenge requires them to look at these test types which could be overwhelming for a beginner maybe?

Reply:
great great feedback: I made it _much_ more hand-hold-y 71a8d2998cd1371dc552423f0bf9724482a3152a

whadday think?